src/mongoose_tls.erl

```diff
@@ -179,7 +180,7 @@ close(#mongoose_tls_socket{tls_module = M, tls_socket = S}) -> M:close(S).
 -spec get_sockmod(socket()) -> module().
 get_sockmod(#mongoose_tls_socket{tls_module = Module}) -> Module.
 
--spec get_tls_last_message(ejabberd_socket:socket()) -> {ok, binary()} | {error, term()}.
+-spec get_tls_last_message(socket()) -> {ok, binary()} | {error, term()}.
```

Comment:
it's quite confusing that we have `socket()` and `tls_socket()` types, and even more confusing that `-type tls_socket() :: any().` and `-type socket() :: #mongoose_tls_socket{}.`

Reply:
extended type